lib/promiscuous_black_hole/embeddings_table.rb

```diff
@@ -0,0 +1,22 @@
+module Promiscuous::BlackHole
+  class EmbeddingsTable
+    def self.dataset
+      DB[:public__embeddings]
+    end
+
+    def self.ensure_exists
+      return if DB.table_exists?(:embeddings, :schema => :public)
+      Locker.new('embeddings').with_lock do
+        DB.create_table?(:public__embeddings) do
+          primary_key [:parent_table, :child_table], :name => :embeddings_pk
+          column :parent_table, 'varchar(255)'
+          column :child_table, 'varchar(255)'
```

Comment:
Is this ever going to run aside from the first time the app is booted up, or if we lost the DB or something? If not, does this belong in an initializer or something instead of here?